app/storybook/documentation/ui-kit/api/CountDistinctMetricQueryInput.mdx

```diff
@@ -0,0 +1,8 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
```

Comment:
Same for other pages
<img width="978" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/f0b9d5e8-e588-4266-ba85-4fbfceebae36">

Reply:
False alarm! Closed.